### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by judithpatudith**
## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-website#testing-your-updates-locally) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-website#contribution-workflow).

### CODE REVIEW

**Comment by renesijnke** [comment]
Hi @judithpatudith ,

I have fixed the issues with the homepage and pushed the fixes directly to the homepage. So all should be back to normal :)

Regards,

Rene

**Comment by judithpatudith** [comment]
Looks great! Thank you!

File: src/index.jade [hunk 1]
```diff
@@ -91,28 +91,36 @@ block content
...
+            p.hero-list__item__copy__paragraph.small Installing distributed data services, databases, CI/CD tools, and monitoring on DC/OS takes just a few clicks or commands.
+            p.
+            a(href='http://universe.dcos.io/#/') Find a service
 
           .lg-col-3.col-6.left-align.px2
-            a(href='https://mesosphere.com/about/').block.m0
-              img.hero-list__item__image(src="/assets/images/icons/mesosphere-gray.svg")
-              h5.hero-list__item__copy__title Commercially backed
-              p.hero-list__item__copy__paragraph.small When you contribute to DC/OS you'll engage with Mesosphere's world-class developers, who help DC/OS grow and thrive every day.
+            // a(href='https://mesosphere.com/about/').block.m0
```
Comment (karlkfi) [inline src/index.jade]: chances of wanting these link blocks back is slim. just nuke the comments

File: src/index.jade [hunk 2]
```diff
@@ -91,28 +91,36 @@ block content
...
+            img.hero-list__item__image(src="/assets/images/icons/containersbigdata.svg")
+            h5.hero-list__item__copy__title Containers & Fast Data
```
Comment (karlkfi) [inline src/index.jade]: Does this single ampersand work? Or does it need to be `&amp;` for html?

File: src/index.jade [hunk 3]
```diff
@@ -150,7 +158,7 @@ block content
         h2 Backed by Mesosphere
         p
           | DC/OS is created and maintained by Mesosphere, Inc., which offers an enterprise version of DC/OS that includes advanced features for security, compliance, multitenancy, networking and storage, and is backed by 24x7, SLA-governed support. For more information about Mesosphere DC/OS Enterprise, please visit
-          a(href='https://mesosphere.com/') mesosphere.com
+          a(href='https://mesosphere.com/')  mesosphere.com
```
Comment (karlkfi) [inline src/index.jade]: this is weird... it shouldnt need a space IN the link... maybe just put the link on the previous line: https://stackoverflow.com/questions/6989653/jade-links-inside-a-paragraph

File: src/index.jade (hunk 2 above)
Comment (judithpatudith) [inline src/index.jade]: Builds fine locally (famous last words)

File: src/index.jade (hunk 3 above)
Comment (judithpatudith) [inline src/index.jade]: Yup! Thanks!

**Comment by judithpatudith** [comment]
@pleia2 I'm pretty sure this is ready, but I'm a bit hesitant to merge it myself since I made the last commit and it's a big change. Can you double check it extra carefully and merge it if you think it's OK?

**Pull request merged**